docs/index.asciidoc

```diff
@@ -88,6 +88,17 @@ will be similar to events directly indexed by {plugin-uc} into Elasticsearch.
 endif::[]
 
 
+[id="plugins-{type}s-{plugin}-memory"]
+===== Memory usage
+
+This plugin uses "off-heap" direct memory in addition to heap memory.
+By default, a JVM's off-heap direct memory limit is the same as the heap size.
+For example, setting `-Xmx10G` without setting the direct memory limit will allocate `10GB` for heap and an additional `10GB` for direct memory, for a total of `20GB` allocated. 
+You can set the amount of direct memory with `-XX:MaxDirectMemorySize` in {logstash-ref}/jvm-settings.html[Logstash's JVM Settings]. Consider setting it to half of the heap size. Setting too low direct memory decreases the performance of ingestion.
+
+NOTE: Be sure that heap and direct memory combined does not exceed the total memory available on the server to avoid OutOfDirectMemoryError
```

Comment:
What about "... to avoid an OutOfDirectMemoryError." (singular) or "... to avoid errors."  (plural)